packages/foreman/foreman/foreman.spec

```diff
@@ -251,79 +251,80 @@ BuildRequires: %{?scl_prefix}rubygem(graphql-batch)
 %if 0%{?scl:1}
 BuildRequires: %{scl}-runtime-assets >= 5
 BuildRequires: %{scl}-runtime-assets < 6
+%else
+BuildRequires: nodejs-packaging
 %endif
-BuildRequires: nodejs >= 6.10
+BuildRequires: %{?scl_prefix_nodejs}nodejs >= 6.10
 BuildRequires: http-parser
 # Temporary dep on libuv until https://bugs.centos.org/view.php?id=10606
 # is resolved
 BuildRequires: libuv
```

Comment:
This bug was filed against CentOS 6 for nodejs. We can probably drop it.

Reply:
seems to work, at least the reproducer in the bug doesn't trigger